### PR TITLE
Create password reset form.

### PR DESCRIPTION
**Pull request by Mariatta**
To replace https://forms.gle/nFVfjgmUwhhY8B3P9

### CODE REVIEW

File: forms/chapter-password-reset.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# PyLadies Email Password Reset Request
+
+Forgot the password to your {chapter}@pyladies.com email address? No worries. PyLadies administrators can help reset the password for you.
```
Comment (lorenanicole) [inline forms/chapter-password-reset.md]: *Suggestion:*
Render `chapter@pyladies.com` as markdown

File: forms/chapter-password-reset.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+For security, we will provide the new password to the organizer that was registered with us.
+If you're an organizer that is not yet registered with us, (for example if you're a new organizer),
+please provide the organizer who we can reach out to to confirm the password reset request.
+
+In case there are multiple organizers registered, we may also contact the other organizers to
+verify the request.
+
+
+- Email Address
```
Comment (lorenanicole) [inline forms/chapter-password-reset.md:15]: Add space

File: forms/chapter-password-reset.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  
+- Your Name (short answer text)
+
+- Email address where the new password will be sent to (short answer text)
+ (if different from above) 
+ 
+- What is the pyladies.com email address that needs reset? (short answer text)
+  Please enter the full email address including domain, Example: vancouver@pyladies.com 
+
+- Additional info (Paragraph)
```
Comment (lorenanicole) [inline forms/chapter-password-reset.md:29]: Add space

File: forms/chapter-password-reset.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Email Address
+  This form is collecting email address
+  
+- Your Name (short answer text)
+
+- Email address where the new password will be sent to (short answer text)
+ (if different from above) 
+ 
+- What is the pyladies.com email address that needs reset? (short answer text)
```
Comment (lorenanicole) [inline forms/chapter-password-reset.md:25]: Add space

File: forms/chapter-password-reset.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+In case there are multiple organizers registered, we may also contact the other organizers to
+verify the request.
+
+
+- Email Address
+  This form is collecting email address
+  
+- Your Name (short answer text)
+
+- Email address where the new password will be sent to (short answer text)
```
Comment (lorenanicole) [inline forms/chapter-password-reset.md:21]: Add space

**Review by lorenanicole — CHANGES_REQUESTED** [review]
some small changes

**Comment by Mariatta** [comment]
Thanks, I've made the suggested changes.

**Review by lorenanicole — APPROVED** [review]
LGTM

**Comment by Mariatta** [comment]
Thanks!

**Pull request merged**